python/cugraph/cugraph/experimental/datasets/dataset.py

```diff
@@ -19,23 +19,30 @@
 
 
 class DefaultDownloadDir:
+    """
+    An object that is used to store the location of the dataset files. It is
+    shared across all dataset objects. Upon creation, it checks for the
+    existence of the environment variable RAPIDS_DATASET_ROOT_DIR. Otherwise,
+    defaults to <HOME>/.cugraph/datasets.
+    """
     def __init__(self):
-        self._dd = Path(os.environ.get("RAPIDS_DATASET_ROOT_DIR",
-                                       Path.home() / ".cugraph/datasets"))
+        self._path = Path(os.environ.get("RAPIDS_DATASET_ROOT_DIR",
+                                         Path.home() / ".cugraph/datasets"))
 
     @property
-    def dd(self):
-        if self._dd is None:
-            self._dd = Path(os.environ.get("RAPIDS_DATASET_ROOT_DIR",
-                                           Path.home() / ".cugraph/datasets"))
-        return self._dd
+    def path(self):
+        if self._path is None:
```

Comment:
Around here could be a good place where you describe how `path` is returned (eg. returns the current setting, and if not set returns the env var RAPIDS_DATASET_ROOT_DIR, else <home>/.cugraph/datasets)
Although, feel free to ignore if you think it's fairly self-documenting as-is.